index.html

```diff
@@ -63,139 +68,152 @@
+              sign-ups required.
+            </p>
+            <ul>
+              <li class="lang">html</li>
+              <li class="lang">css</li>
+              <li class="lang">javaScript</li>
+            </ul>
+            <a class="moreabout" href="#">See Project</a>
+          </div>
         </div>
 
         <!-- 3rd work! -->
         <div class="works">
           <img src="images/works/work3.svg" alt="" />
-          <h2>Tonic</h2>
-          <ul>
-            <li class="worksubtitle">CANOPY</li>
-            <li class="bullets">Back End Dev</li>
-            <li class="bullets">2015</li>
-          </ul>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-          <ul>
-            <li class="lang">html</li>
-            <li class="lang">css</li>
-            <li class="lang">javaScript</li>
-          </ul>
-          <a class="moreabout" href="#">See Project</a>
+          <div class="works-desk">
+            <h2>Tonic</h2>
+            <ul>
+              <li class="worksubtitle">CANOPY</li>
+              <li class="bullets">Back End Dev</li>
+              <li class="bullets">2015</li>
+            </ul>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul>
+              <li class="lang">html</li>
+              <li class="lang">css</li>
+              <li class="lang">javaScript</li>
+            </ul>
+            <a class="moreabout" href="#">See Project</a>
+        </div>
         </div>
 
         <!-- 4rt work! -->
         <div class="works">
           <img src="images/works/work4.svg" alt="" />
-          <h2>Tonic</h2>
-          <ul>
-            <li class="worksubtitle">CANOPY</li>
-            <li class="bullets">Back End Dev</li>
-            <li class="bullets">2015</li>
-          </ul>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-          <ul>
-            <li class="lang">html</li>
-            <li class="lang">css</li>
-            <li class="lang">javaScript</li>
-          </ul>
-          <a class="moreabout" href="#">See Project</a>
+          <div class="works-desk">
+            <h2>Tonic</h2>
+            <ul>
+              <li class="worksubtitle">CANOPY</li>
+              <li class="bullets">Back End Dev</li>
+              <li class="bullets">2015</li>
+            </ul>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul>
+              <li class="lang">html</li>
+              <li class="lang">css</li>
+              <li class="lang">javaScript</li>
+            </ul>
+            <a class="moreabout" href="#">See Project</a>
+        </div>
```

Comment:
- [ ] Kindly follow the Figma guidelines, design and fix the cards. As you can see all the cards in even numbers are flex reverse where Multi post stories are placed first and the image second. Fix it.
- Your design:
<img width="976" alt="Screenshot 2022-03-27 at 12 07 28 AM" src="https://user-images.githubusercontent.com/70269507/160257186-5c552fb9-3b7a-4c61-ac42-4e1aea186b2b.png">
<img width="976" alt="Screenshot 2022-03-27 at 12 07 33 AM" src="https://user-images.githubusercontent.com/70269507/160257189-9645cab0-3319-4892-8b03-315978b7c30a.png">

Figma design:
<img width="354" alt="Screenshot 2022-03-27 at 12 07 03 AM" src="https://user-images.githubusercontent.com/70269507/160257210-0d839999-5de2-4530-b508-6bf24d7ed258.png">